src/main/resources/com/google/api/codegen/java/discovery_fragment.snip

```diff
@@ -66,11 +66,18 @@
         JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
 
         {@ApiName} {@serviceName()} = new {@ApiName}.Builder(httpTransport, jsonFactory, credential)
-          .setApplicationName("Google Cloud Platform Sample")
-          .build();
+            .setApplicationName("Google Cloud Platform Sample")
+            .build();
 
         {@initialize(method)}
         {@methodCall(method)}
+        @if context.hasMediaDownload(method)
+
+          // {@TODO()} Download desired media content. For more information, see:
```

Comment:
Downloading media content is not the only use case for such methods, as their generic form is used to download metadata. So the user only need add code to download media content if desired. I'm open to alternative suggestions on wording.